### PR TITLE
finish mobile version portfolio

### PR DESCRIPTION
**Pull request by lawrahkonwea**
Description of the project. This is the second project of the mobile version portfolio, 
- Html used
- Cascading style sheet (css) used

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -48,7 +48,184 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
...
+                <button class="hypertext">html</button>
+                <button class="cascading">css</button>
+                <button class="javascript">javaScript</button>
+            </div>
+            <div>
+                <button class="project-btn"> <a href="#">See Project</a></button>
+            </div>
+           </div>
+        </main>
+        </section>
```
Comment (Santiago220991) [inline index.html]: - Kindly wrap all the work cards inside a `section` tag. It represents a generic standalone section of the document, so it should be wrapped inside a `section` tag. :nerd_face: :nerd_face: Also please use the display:grid property to display all the cards.  
- Every card should be inside a `div` tag. Kindly fix this. :pray: :pray:

File: index.html [hunk 2]
```diff
@@ -48,7 +48,184 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
...
-        </section>
+      </section>
     </main>
+    
+    <section class="project-main-wrapper">
+        <div class="project-container">
+            <div class="postcollection">
+            <div class="img-container"><img class="snapshoot-img" src="images/Snapshoot Portfolio.png"alt="snapshot portfolio 1"></div>
+            <div class="project-review">
+            <label>Tonic</label>
```
Comment (Santiago220991) [inline index.html]: Kindly use an `hx` tag for this subtitle. Apply this change on every card. :pray: :pray:

File: index.html [hunk 3]
```diff
@@ -48,7 +48,184 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
...
+                <div><p class="backend">2015</p></div>
+            </div>
+            <div>
+                <p class="access-project">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="programming-lang">
+                <button class="hypertext">html</button>
+                <button class="cascading">css</button>
+                <button class="javascript">javaScript</button>
+            </div>
```
Comment (Santiago220991) [inline index.html]: Please replace the <div class="programming-lang"> for a `ul` tag and each technology should be inside a `li` tag. This is for semantics reasons. Kindly apply this change to every card. :+1: :smiley: :smiley:

File: index.html [hunk 4]
```diff
@@ -48,7 +48,184 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
...
+        </main>
+        </section>
+
+              
+            <section class="containerr">
+              <div class="project-main-wrapperrr">
+                <div class="content-wrapperrr">
+                 <div class="main-content">
+                    <div class="about-me">
+                        <label>About Myself</label>
```
Comment (Santiago220991) [inline index.html]: Kindly use an `hx` tag for this subtitle. :pray: :pray:

File: style.css [hunk 5]
```diff
@@ -131,5 +145,190 @@ h1 {
...
   width: 50%;
-  margin-bottom: 40%;
+  margin-bottom: 30%;
+}
+
+.project-main-wrapper {
+  background-color: #fff;
+  border-radius: 20px;
+  margin: auto 27px;
+}
```
Comment (Santiago220991) [inline style.css]: Kindly make sure that your project sticks as much as possible to the Figma template provided. Kindly increase the top margin of your image card. You could get this value from the Figma template provided. :eyes: :eyes: 
Your design
image

Figma design
image

File: style.css [hunk 6]
```diff
@@ -131,5 +145,190 @@ h1 {
...
+  line-height: 52px;
+  color: #172b4d;
+  margin-top: 20px;
+}
+
+.project-main-wrapperrr {
+  background-color: #fff;
+  margin-top: 60px;
+  border-top-right-radius: 80px;
+}
```
Comment (Santiago220991) [inline style.css]: Kindly make sure that your project sticks as much as possible to the Figma template provided. Kindly remove the top-right-border radius property from this container. :eyes: :eyes: 
Your design
image

Figma design
image

**Review by Santiago220991 — CHANGES_REQUESTED** [review]
Hi  @lawrahkonwea, 
I'm Santiago your reviewer for this PR. :wave: :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
:heavy_check_mark: Good PR summary.
:heavy_check_mark: Followed GitHub flow.
:heavy_check_mark: No linter errors.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -48,7 +48,189 @@ <h3 class="lets-connect">LET'S CONNECT</h3>
...
+                    <div class="programming-lang">
+                        <ul>
+                        <li class="lang">html</li>
+                        <li class="lang">css</li>
+                        <li class="lang">javaScript</li>
+                        </ul>
+                    </div>
+                    <div>
+                        <button class="project-btn"> <a href="#">See Project</a></button>
+                    </div>
```
Comment (Debas-31) [inline index.html:173]: - [ ] Kindly implement the display technology for CSS. The previous CR asked to change the project requirement to use Grid and flex. For the projects container properties ` display: Grid` and for project container properties  `display: flex`. Please fix it :wrench:

**Review by Debas-31 — CHANGES_REQUESTED** [review]
> ## Hi  @lawrahkonwea  :wave:
# Status: Required Change :recycle:
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters setup :heavy_check_mark:
- Github flow :heavy_check_mark: 
- Project working :heavy_check_mark:  
 
## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Favourezeugwa — APPROVED** [review]
## STATUS: APRROVED 🏆 🍹 🥇 

Hi @lawrahkonwea , 

Great Job implementing all the required changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

happy

## Highlights 💯 
- No linter errors ✔️ 
- Descriptive commit messages ✔️ 
- Used Github flow ✔️ 
- Professional Readme ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**